internal/cephfs/util.go

```diff
@@ -32,6 +32,15 @@ import (
 	"k8s.io/klog"
 )
 
+// enum defining logging levels.
+const (
+	One klog.Level = iota + 1
+	Two
```

Comment:
The names can be as follow
Default
Useful
Extended
Debug
Trace 

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md we can change to meaningful names

Reply:
Sure. That makes more sense. Will change them.